deploy/pdf-blueprint.ipynb

```diff
@@ -101,16 +101,16 @@
    "source": [
     "## Spin up the blueprint\n",
     "\n",
-    "NOTE: This step can take about 10 minutes. The following steps are how we suggest running and monitoring your the progress. If you do not want to monitor - you can simply run `docker compose --profile retrieval up --build -d` in a new terminal window\n",
+    "NOTE: This step can take about 10 minutes. The following steps are how we suggest running and monitoring your the progress. If you do not want to monitor - you can simply run `docker compose --profile retrieval --profile table-structure up -d` in a new terminal window\n",
```

Comment:
Since this will be going on a release branch we remove the `--build` option and simply download the release container. This also speeds up startup time significantly.